siff-dr/siff-header.go

```diff
@@ -10,8 +10,8 @@ import (
 
 // SIFF constants
 const (
-	EXP  layers.IPv4Flag = 1 << 3
-	EVIL layers.IPv4Flag = 1 << 2 // http://tools.ietf.org/html/rfc3514 ;)
+	EXP  layers.IPv4Flag = 1 << 2
+	EVIL layers.IPv4Flag = 1 << 3 // http://tools.ietf.org/html/rfc3514 ;)
```

Comment:
Can you rename these to be in [mixed caps](https://golang.org/doc/effective_go.html#mixed-caps)?

Also, how is there even `1 << 3` as an ip flag?  The IPv4 flags field is only three bits.